cscript, wscript: add page; ospp.vbs, slmgr.vbs: add cscript/wscript notice (#22064)
tldr-pages/tldr: pages/windows/ospp.vbs.md

# ospp.vbs

> Install, activate, and manage volume licensed versions of Microsoft Office products.

> > Use `cscript` to run this program as a CLI, or `wscript` as a GUI.

Reviewer: ```suggestion
> Use `cscript` to run this program in the CLI, or `wscript` in the GUI.
```
